src/rails.js

```diff
@@ -115,7 +115,7 @@
         if (element.is('form')) {
           method = element.attr('method');
           url = element.attr('action');
-          data = element.serializeArray();
+          data = data = $(element[0].elements).serializeArray();
```

Comment:
@rafaelfranca - Typo. I can't see any legitimate reason I'd have done that on purpose. I can fix that up, along with anything else you spot.